packages/asset/contracts/AssetReveal.sol

```diff
@@ -18,11 +18,11 @@ contract AssetReveal is IAssetReveal, Initializable, ERC2771Handler, EIP712Upgra
     AuthValidator private authValidator;
 
     // mapping of creator to asset id to asset's reveal nonce
-    mapping(address => mapping(uint256 => uint16)) revealIds;
+    mapping(address => mapping(uint256 => uint16)) public revealIds;
```

Comment:
Why do we want to make this public?

Reply:
We could make it internal or private but it was public as we didn't define its visibility

Reply:
This is actually not the case here, a mapping is `internal` by default, so you are making it public here.